[CONTENT REVIEW] 21.4 - Adicionar requisito de Trigger ao projeto Vocabulary Booster + Novo padrão readme (#6)
betrybe/sd-0x-mysql-vocabulary-booster: README.md

---

## Data de Entrega

  - Projeto individual.

>   - Serão dois dias de projeto.

Reviewer: No calendário o Vocabulary Booster continua sendo apenas um dia de projeto. @roziscoding @mariocmachado vai realmente ser dois dias a partir da T6?

Reviewer 2: @roziscoding não entendi também essa questão dos 2 dias. Atualmente o calendário preve 1 dia apenas

Reviewer 2: @orlando-messias você lembra onde foi passado essa questão do número de dias do projeto?

Reviewer 3: preciso da confirmação para alterar e fazer o merge, @roziscoding , acabei de lembrar do requiriments.json e já comitei

Reviewer 3: ```suggestion
  - Será um dia de projeto.
```
Posso :shipit: ?

      * Exemplo:
        * `git add .` (adicionando solução para desafio 1)
        * `git status` (deve aparecer listado o arquivo _desafio1.sql_ em verde)
  * Faça o `commit` inicial
      * Exemplo:
        * `git commit -m 'iniciando o projeto MySQL Vocabulary Booster'` (fazendo o primeiro commit)
        * `git status` (deve aparecer uma mensagem tipo _nothing to commit_ )

---

### DURANTE O DESENVOLVIMENTO

* Faça `commits` das alterações que você fizer no código regularmente

* Lembre-se de sempre após um (ou alguns) `commits` atualizar o repositório remoto

* Os comandos que você utilizará com mais frequência são:
  1. `git status` _(para verificar o que está em vermelho - fora do stage - e o que está em verde - no stage)_
  2. `git add` _(para adicionar arquivos ao stage do Git)_
  3. `git commit` _(para criar um commit com os arquivos que estão no stage do Git)_
  4. `git push -u nome-da-branch` _(para enviar o commit para o repositório remoto na primeira vez que fizer o `push` de uma nova branch)_
  5. `git push` _(para enviar o commit para o repositório remoto após o passo anterior)_

---

### DEPOIS DE TERMINAR O DESENVOLVIMENTO (OPCIONAL)

> Para sinalizar que o seu projeto está pronto para o _"Code Review"_ dos seus colegas, faça o seguinte:

Reviewer 4: ```suggestion-
Para sinalizar que o seu projeto está pronto para o _"Code Review"_ de colegas, faça o seguinte:
```

Reviewer 4: LNG :)

> * Vá até a página **DO SEU** _Pull Request_, adicione a label de _"code-review"_ e marque seus colegas:

Reviewer 4: ```suggestion
* Vá até a página **DO SEU** _Pull Request_, adicione a label de _"code-review"_ e marque as pessoas de quem quer receber o review:
```

  * No menu à direita, clique no _link_ **"Labels"** e escolha a _label_ **code-review**;

  * No menu à direita, clique no _link_ **"Assignees"** e escolha **o seu usuário**;

  * No menu à direita, clique no _link_ **"Reviewers"** e digite `students`, selecione o time `tryber/students-sd-0x`.

Caso tenha alguma dúvida, [aqui tem um video explicativo](https://vimeo.com/362189205).

---

> ## Linter

Reviewer 4: Temos mesmo ESLint pra esse projeto? Ele não é todo em SQL?

Reviewer: @roziscoding é apenas SQL mesmo nesse projeto. Acho que podemos remover esse trecho.

>Use o banco `w3schools` como referência

Monte uma query que exiba 03 colunas:

1. A primeira deve possuir o alias "**Nome de contato**" e exibir **o nome de contato** da pessoa consumidora.

2. A segunda deve possuir o alias "**Empresa que fez o envio**" e exibir o nome da empresa que efetuou o envio do pedido.

3. A terceira deve possuir o alias "**Data do pedido**" e exibir a data que o pedido foi feito.

Seus resultados devem estar ordenados pelo nome de contato da pessoa consumidora em ordem alfabética.

Em caso de empate no nome de contato, ordene os resultados pelo nome da empresa que fez o envio do produto em ordem alfabética.

Se houver empresas com o mesmo nome, ordene os resultados pela data do pedido em ordem crescente.

---

## Desafio 9

Exiba todas as pessoas funcionárias que já realizaram algum pedido e o total dos pedidos feitos.

>Use o banco `w3schools` como referência

Monte uma query que exiba 02 colunas:

1. A primeira deve possuir o alias "**Nome completo**" e exibir o nome completo da pessoa funcionária (não se esqueça do espaço entre o nome e o sobrenome).

> 2. A segundo deve possuir o alias "**Total de pedidos**" e exibir a quantidade total de pedidos feitos pela pessoa.

Reviewer 4: ```suggestion
2. A segunda deve possuir o alias "**Total de pedidos**" e exibir a quantidade total de pedidos feitos pela pessoa.
```
